soccer/radio/Radio.hpp

```diff
@@ -17,7 +17,8 @@ class Radio {
     Radio() { _channel = 0; }
 
     virtual bool isOpen() const = 0;
-    virtual void send(Packet::RadioTx& packet) = 0;
+    virtual void send(Packet::RadioTx& radioTx,
+                      const std::array<RobotIntent, Num_Shells>& intents) = 0;
```

Comment:
I don't know if the cleanest way to do this logging is by passing it out like this...I need to think some more about how we want to do logging, I think, but this should be fine for this PR.